### PR TITLE
Split keywords by full-width space

### PR DESCRIPTION
**Pull request by myokoym**
Close bm-sms/nursepress/issues/166

### CODE REVIEW

File: app/groonga/post_searcher.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def search(query)
           (target.index("Terms.Posts_title") * 10) |
             target.index("Terms.Posts_content")
         end
-        keywords = query.keywords.split
+        keywords = query.keywords.gsub(/　/, " ").split
```
Comment (okkez) [inline app/groonga/post_searcher.rb]: How about `query.keywords.split(/[[:blank:]]/)` ?

See http://docs.ruby-lang.org/ja/2.3.0/doc/spec=2fregexp.html

Comment (myokoym) [inline app/groonga/post_searcher.rb]: Oh...it's reasonable. I've fixed it.

File: test/integration/search_test.rb [hunk 2]
```diff
@@ -106,6 +106,28 @@ class SearchTest < ActionDispatch::IntegrationTest
     end
   end
 
+  test "full-width space should be delimiter" do
```
Comment (tricknotes) [inline test/integration/search_test.rb]: I think such test case should be a model test for performance.

File: test/models/post_searcher_test.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    @site = create(:site)
+    @indexer = PostIndexer.new
+  end
+
+  teardown do
+    teardown_groonga_database
+  end
+
+  test "full-width space should be delimiter" do
+    post = create_post(
```
Comment (myokoym) [inline test/models/post_searcher_test.rb]: Thanks. I've fixed it.

File: test/integration/search_test.rb [hunk 4]
```diff
@@ -106,6 +106,28 @@ class SearchTest < ActionDispatch::IntegrationTest
...
+
+    visit "/"
+
+    fill_in "query[keywords]", with: keywords
+
+    click_on "検索"
+
+    within("main") do
+      # NOTE: Capybara::Node::Finders#find may replace full-width space with half-width space...
+      normalized_message = find(".message").text.gsub(/ /, "　")
```
Comment (myokoym) [inline test/integration/search_test.rb]: I've removed this code by  2211c8c373979269415d17acc5575c400c4aea99

**Pull request merged**